documentation/docs/steps/checkChangeInDevelopment.md

```diff
@@ -4,7 +4,22 @@
 
 ## Prerequisites
 
-* No prerequisites
+* For type SOLMAN
+
+| SOLMAN Version | cm client version |
+|-----------------------------------------|----------------|
+|SAP Solution Manager 7.2 SP6, SP7        | cm_client v1.x |
+|SAP Solution Manager 7.2 SP 8 and higher | cm_client v2.0 |
```

Comment:
We should mention how to configure the docker image for cm client v1. --> will add another commit for that.